index.html

```diff
@@ -44,15 +67,20 @@ <h5 class="supporting-text">I’am a Software Developer</h5>
 
                 <section>
                     <div class="works-section">
-                        <h1 class="title">My Recent Works</h1>
+                        <h1 class="title">My Recent Works </h1>
```

Comment:
- [OPTIONAL] Kindly make sure your work section is centered as per the Figma design:
- [OPTIONAL] Kindly make sure there is a hover effect to these cards as show in the design:

![Screenshot from 2023-01-21 14-38-39](https://user-images.githubusercontent.com/97834160/213868171-85818219-bb03-4e4f-accc-764afb528d7d.png)